.github/workflows/ci-workflow.yml

```
@@ -33,3 +33,19 @@ jobs:
         run: npm clean-install
       - name: Run integration tests
         run: npm run integration-test
+  code_coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
```

Il est préférable d'utiliser la même version de *Node.js* que les jobs ci-dessus : 
```suggestion
          node-version: 12.16.1
```

---

ok corrigé